### DISCUSSION TITLE
Subtitles and Extractor extension

### DISCUSSION THREAD

**Discussion by ProgramComputer** [HugoFara/lwt#188] [Show and tell]
This show and tell will display the features from https://github.com/HugoFara/lwt/pull/157. You can extract official subtitles from a web source in text import. The audio will play in test mode for sentences.

Turn on audio when playing video.

https://github.com/HugoFara/lwt/assets/22284856/3dcbd6d1-dacc-4ee7-a0ec-7179ecdc3a94

**Comment by HugoFara** [comment]
Great work!
